src/Bundle/Controller/ResourceController.php

```diff
@@ -240,7 +276,11 @@ public function updateAction(Request $request): Response
         $this->isGrantedOr403($configuration, ResourceActions::UPDATE);
         $resource = $this->findOr404($configuration);
 
-        $form = $this->resourceFormFactory->create($configuration, $resource);
+        if (null !== $input = $configuration->getInput()) {
+            $resource = $this->transform($resource, $input, $configuration);
```

Comment:
I think the `assertNotNull' is missing

Reply:
Hum this is the same as `createAction`
to update the password, for example, we do not want to instanciate any transformation and pass null data to the form.